src/Libraries/Microsoft.Extensions.AI/ChatCompletion/DistributedCachingChatClient.cs

```diff
@@ -59,9 +56,9 @@ public DistributedCachingChatClient(IChatClient innerClient, IDistributedCache s
     /// <summary>Gets or sets JSON serialization options to use when serializing cache data.</summary>
     public JsonSerializerOptions JsonSerializerOptions
     {
-        get => _jsonSerializerOptions;
-        set => _jsonSerializerOptions = Throw.IfNull(value);
-    }
+        get;
+        set => field = Throw.IfNull(value);
+    } = AIJsonUtilities.DefaultOptions;
```

Comment:
Did you manually fix this one up? I was getting this on a separate line when using `dotnet format`.